pyshelf/bulk_update/runner.py

```diff
@@ -49,11 +49,10 @@ def _run_process(self, config):
 
     def _find_bucket_list(self, requested_bucket_list=None):
         bucket_list = []
-        for name, bucket_config in self.all_buckets.iteritems():
+        for bucket in self.all_buckets:
```

Comment:
Could you rename to `all_bucket_list` since it is now a list instead of a structure?